lib/_emerge/actions.py

```diff
@@ -432,17 +430,18 @@ def action_build(
             # instances need to load remote metadata if --getbinpkg
             # is enabled. Use getbinpkg_refresh=False to use cached
             # metadata, since the cache is already fresh.
-            if "--getbinpkg" in emerge_config.opts or quickpkg_direct:
+            if quickpkg_direct:
```

Comment:
In this particular case it is necessary to call populate for --getbinpkg because there is a freshly instantiated binarytree which will not contain remote packages unless we call populate with getbinpkgs=True.